core/src/main/java/org/testcontainers/containers/wait/strategy/HostPortWaitStrategy.java

```diff
@@ -25,7 +25,9 @@
     protected void waitUntilReady() {
         final Set<Integer> externalLivenessCheckPorts = getLivenessCheckPorts();
         if (externalLivenessCheckPorts.isEmpty()) {
-            log.debug("Liveness check ports of {} is empty. Not waiting.", waitStrategyTarget.getContainerInfo().getName());
+            if (log.isDebugEnabled()) {
```

Comment:
`waitStrategyTarget.getContainerInfo()` will call Docker API in case of Docker Compose, and, if debug is disabled, it is unnecessary